sqlitecluster/SQLiteNode.h

```diff
@@ -203,6 +203,15 @@ class SQLiteNode : public STCPNode {
     void setQuroumCheckpoint(const int quroumCheckpoint) { _quorumCheckpoint = quroumCheckpoint; };
     int getQuorumCheckpoint() { return _quorumCheckpoint; };
 
+    // The peer we should sync from is recalculated every time we call this. If no other peer is logged in, or no
+    // logged in peer has a higher commitCount that we do, this will return null.
+    Peer* updateSyncPeer();
```

Comment:
Is this intentionally part of the "external/public API"?  If it's only intended for internal use, can you put this into the `private:` section and prefix with an underscore?